app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx

```diff
@@ -162,7 +162,7 @@ export class SimpleWidget extends Component {
   getWidgetContent = () => {
     const { widgetType, isPrintMode } = this.props;
     const { widget, queryParameters, userSettings } = this.state;
-
+    const isTestCaseSearch = widgetType === TEST_CASE_SEARCH;
```

Comment:
Maybe introduce the new map for such widgets?
Like `HANDLES_EMPTY_STATE_INTERNALLY_WIDGETS` or smth like that.